nx-X11/programs/Xserver/hw/nxagent/Options.h

```diff
@@ -384,6 +384,9 @@ typedef struct _AgentOptions
 
   /*
    * Inhibit some XKEYBOARD requests.
+   * 0 = allow all XKEYBOARD requests
+   * 1 = forbid all XKEYBOARD requests if keyboard=clone or keyboard=query
```

Comment:
I wonder if `InhibitXkb` is actually doing what it's supposed to do? We should see denied change requests a lot more often if it were working correctly.